kili/project/__init__.py

```diff
@@ -27,8 +27,8 @@ def __init__(  # pylint: disable=too-many-arguments
     @typechecked
     def export(  # pylint: disable=too-many-arguments
         self,
-        output_filename: str,
-        output_format: LabelFormat,
+        filename: str,
+        fmt: LabelFormat,
```

Comment:
These arguments appear to me to be easier to use because shorter, without making any confusion for the end user. 
Could be discussed if you disagree.
I used this on the SDK V2 function and updated the one on the SDK V3

Reply:
👍